.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+Fixes #
+<!-- Link to relevant issue (for ex: #1234) which will automatically close the issue once the PR is merged -->
+
+## PR Type
+What kind of change does this PR introduce?
+<!-- Please uncomment one ore more that apply to this PR -->
+
+<!-- - Bugfix -->
+<!-- - Feature -->
+<!-- - Code style update (formatting) -->
+<!-- - Refactoring (no functional changes, no api changes) -->
+<!-- - Build or CI related changes -->
+<!-- - Documentation content changes -->
+<!-- - Sample app changes -->
+<!-- - Other... Please describe: -->
+
+
+## What is the current behavior?
+<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
+
+
+## What is the new behavior?
+
+
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tested code with current [supported SDKs](../readme.md#supported)
+- [ ] Pull Request has been submitted to the documentation repository [instructions](..\contributing.md#docs). Link: <!-- docs PR link -->
```

Comment:
Think some of these links are from the other repo, so we probably need to update them or add more guidance here for this repo?